nginx/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# a minimal Nginx container including containerbuddy and a simple virtulhost config
+FROM nginx:latest
```

Comment:
The Containerbuddy configuration is embedded in the container build, remember? So the Cloudflare container that results from this is a demo -- the whole thing is a demo and doesn't stand alone.